tools/ctl/linux/aws-otel-collector-ctl

```diff
@@ -44,20 +44,41 @@ UsageString="
   status:                                 get the status of the agent process.
 
   -c: configuration
-  file:<file-path>:                       file path on the host
+  <config-uri>:                           - file path on the host. E.g.: /tmp/config.yaml
```

Comment:
```suggestion
  <config-uri>:                           - file path on the host. E.g.: file:/tmp/config.yaml
```
Is the conf map provider required for file uris still?

Reply:
no. that is optional.